requirements.txt

```diff
@@ -1,7 +1,11 @@
+dataclass-wizard
```

Comment:
Following the changes in the lib, the requirements here will also need to change.

Reply:
Yes, removed.